src/Sentry.Serilog/SentrySinkExtensions.cs

```diff
@@ -83,9 +81,9 @@ public static class SentrySinkExtensions
     /// </example>
     public static LoggerConfiguration Sentry(
         this LoggerSinkConfiguration loggerConfiguration,
-        string? dsn = null,
-        LogEventLevel minimumBreadcrumbLevel = LogEventLevel.Information,
-        LogEventLevel minimumEventLevel = LogEventLevel.Error,
+        string dsn,
+        LogEventLevel? minimumBreadcrumbLevel = null,
+        LogEventLevel? minimumEventLevel = null,
```

Comment:
Why are we doing away with the default values?

Reply:
They have no impact... They're the same as the default values for those properties when you create the new SentrySerilogOptions.